rosdep/python.yaml

```diff
@@ -3160,6 +3160,7 @@ python-progressbar:
     artful: [python-progressbar]
     bionic: [python-progressbar]
     cosmic: [python-progressbar]
+    focal: [python3-progressbar]
```

Comment:
I think it would be better to add a new key for the python3 version of the package, [per our guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-3).